test/conformance/conformance_parallel_reduce.cpp

```diff
@@ -56,6 +57,37 @@ struct ReduceBody {
     }
 };
 
+template <class T>
```

Comment:
`//!` explaining what this struct is for?

Reply:
Done